content/wochenplan.pug

```diff
@@ -84,102 +97,110 @@ block main
-			+Ort: +extlink('https://o-phase.live') o-phase.live
-		+Event('16:00', 'Sprechstunde')
-			| Deine Möglichkeit, alle Fragen zu stellen, die dir auf der Seele brennen!
-			+Ort Discord
-
-	+Tag('Montag', '26. Oktober 2020')
-		+Event('14:00', 'FBI M.Sc. Informatik')
-			| Die Fachbereichsinformation für alle, die Informatik im Master studieren. Hier erfährst du, wie dein Studium abläuft!
-			+Ort: +extlink('https://o-phase.live') o-phase.live
-		+Event('16:00', 'Distributed Kochen')
-			+Ort Discord
-		+Event('18:00', 'Sprechstunde')
-			| Deine Möglichkeit, alle Fragen zu stellen, die dir auf der Seele brennen!
-			+Ort Discord
-
-	+Tag('Dienstag', '27. Oktober 2020')
-		+Event('10:00', 'Sprechstunde')
-			| Deine Möglichkeit, alle Fragen zu stellen, die dir auf der Seele brennen!
-			+Ort Discord
-		+Event('14:00', 'FBI M.Sc. Mathematik')
-			| Die Fachbereichsinformation für alle, die Mathematik im Master studieren. Hier erfährst du, wie dein Studium abläuft!
-			+Ort: +extlink('https://o-phase.live') o-phase.live
-
-	+Tag('Mittwoch', '28. Oktober 2020')
-		+Event('14:00', 'FBI B.Sc. Mathematik')
-			| Die Fachbereichsinformation für alle, die Mathematik im Bachelor studieren. Hier erfährst du, wie dein Studium abläuft!
-			+Ort: +extlink('https://o-phase.live') o-phase.live
-		+Event('15:00', 'Online-Spielenachmittag & -abend')
-			+Ort Discord
-		+Event('16:00', 'Sprechstunde')
-			| Deine Möglichkeit, alle Fragen zu stellen, die dir auf der Seele brennen!
-			+Ort Discord
-
-	+Tag('Donnerstag', '29. Oktober 2020')
-		+Event('14:00', 'FBI B.Sc. Informatik')
-			| Die Fachbereichsinformation für alle, die Informatik im Bachelor studieren. Hier erfährst du, wie dein Studium abläuft!
-			+Ort: +extlink('https://o-phase.live') o-phase.live
-		+Event('14:30', 'Campusführung')
-			+Ort Discord
-		+Event('18:00', 'Sprechstunde')
-			| Deine Möglichkeit, alle Fragen zu stellen, die dir auf der Seele brennen!
-			+Ort Discord
-
-	+Tag('Freitag', '30. Oktober 2020')
-		+Event('10:00', 'Sprechstunde')
-			| Deine Möglichkeit, alle Fragen zu stellen, die dir auf der Seele brennen!
-			+Ort Discord
-		+Event('13:30', 'Abschlussveranstaltung')
-			| Die gemeinsame Abschlussveranstaltung von Studier Langsam.
-			+Ort Twitch
-		+Event('14:00', 'Abschlussveranstaltung der Fachschaft')
-			| Die offizielle Abschlussveranstaltung der Fachschafte Mathematik und Informatik.
-			+Ort: +extlink('https://o-phase.live') o-phase.live
+			+Ort #[+Kartenlink(Karte_Audimax) Audimax] und #[+extlink('https://o-phase.live') online]
+
+	+Tag('Montag', '04./11. Oktober 2021')
+		+Event('10:00', 'Kennenlernen')
+			| Ihr kennt uns nicht, wir kennen euch nicht, ihr kennt euch nicht gegenseitig. Lasst uns das ändern, denn Studieren ist ein Mannschaftssport!
+			+Ort Treffpunkt: Vor dem #[+Kartenlink(Karte_Infobau) Infobau]
+		+Event('12:30', 'Pizza-Mittag')
+			| Gemeinsames füllen der leeren Mägen mit Karlsruhes besten Pizzen :)
```

Comment:
You could use a proper emoji here, using `#[+emoji('<your emoji here>')]`. Looks more professional imo.